src/modules/shared/components/ShareFolderBlade/ShareFolderBlade.tsx

```diff
@@ -0,0 +1,161 @@
+import { CampaignMonitorService } from '@shared/services/campaign-monitor-service/campaign-monitor.service';
+import { toastService } from '@shared/services/toast-service';
+
+import { labelKeys, SHARE_FOLDER_FORM_SCHEMA } from './ShareFolderBlade.consts';
+import styles from './ShareFolderBlade.module.scss';
+import { ShareFolderBladeFormState, ShareFolderBladeProps } from './ShareFolderBlade.types';
+
+const ShareFolderBlade: FC<ShareFolderBladeProps> = ({ isOpen, onClose, folderId, folderName }) => {
+	const { tText } = useTranslation();
+
+	const user = useSelector(selectUser);
+	const [emailInputValue, setEmailInputValue] = useState('');
+
+	const link = `${window.location.origin}${ROUTES.shareFolder.replace(':id', folderId)}`;
+
+	const {
+		setValue,
+		handleSubmit,
+		formState: { errors },
+		control,
+	} = useForm<ShareFolderBladeFormState>({
+		resolver: yupResolver(SHARE_FOLDER_FORM_SCHEMA()),
+	});
+
+	const handleClose = () => {
+		setEmailInputValue('');
+		setValue('email', '');
+		onClose();
+	};
+
+	const handleSend = async () => {
+		try {
+			if (user) {
+				await CampaignMonitorService.send({
+					template: 'shareFolder',
+					data: {
+						to: emailInputValue,
+						consentToTrack: 'unchanged',
+						data: {
+							sharer_email: user.email,
+							sharer_name: user.fullName,
+							folder_name: folderName,
+							folder_sharelink: link,
+						},
+					},
+				});
+				toastService.notify({
+					maxLines: 3,
+					title: tText('pages/account/map-delen/folder-id/index___map-is-gedeeld'),
+					description: `${tText(
+						'pages/account/map-delen/folder-id/index___map-is-verstuurd-naar'
+					)} ${emailInputValue}`,
+				});
+				handleClose();
+			}
+		} catch (err) {
+			console.error(err);
+			toastService.notify({
+				maxLines: 3,
+				title: tText('pages/account/map-delen/folder-id/index___error'),
+				description: tText(
+					'pages/account/map-delen/folder-id/index___error-er-is-iets-misgelopen'
+				),
+			});
+		}
+	};
+
+	const renderTextInput = (field: ControllerRenderProps<ShareFolderBladeFormState, 'email'>) => {
+		setEmailInputValue(field.value);
+		return <TextInput {...field} id={labelKeys.email} />;
+	};
+
+	const renderFooter = () => {
+		return (
+			<div className={styles['c-share-folder-blade__close-button-container']}>
+				<Button
+					label={tText('pages/account/map-delen/folder-id/index___sluit')}
+					variants={['block', 'text', 'dark']}
+					onClick={onClose}
+					className={styles['c-share-folder-blade__close-button']}
+				/>
+			</div>
+		);
+	};
+
+	return (
+		<Blade
+			isOpen={isOpen}
+			renderTitle={() => (
+				<h3 className={styles['c-share-folder-blade__title']}>
+					{tText('pages/account/map-delen/folder-id/index___deel-map')}
+				</h3>
+			)}
+			footer={isOpen && renderFooter()}
+			onClose={handleClose}
+		>
+			<div className={styles['c-share-folder-blade__content']}>
+				<>
+					<dt className={styles['c-share-folder-blade__content-label']}>
```

Comment:
`<dt>` and `<dd>` are part of a definition list `<dl>` so if you want to use this correctly, you need to wrap these into `<dl>` (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl).

On another note, I don't think it's semantically correct if we use these tags for a form. I think there are better html tags for this such as `label` for the labels. The FormControl already contains an `input` so i think you can wrap those in divs if needed.